tests/api2/test_012_api_key.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+# Author: Eric Turgeon
+# License: BSD
+# Location for tests into REST API of FreeNAS
+
+import contextlib
+import io
+import json
+import os
+import sys
+
+import requests
+
+apifolder = os.getcwd()
+sys.path.append(apifolder)
+from functions import POST, GET, DELETE, PUT, SSH_TEST, wait_on_job
+from auto_config import pool_name, scale, ha, password, user, ip
+
```

Comment:
Is this a SCALE-only feature? If so, you may want to skip this test if not SCALE

Reply:
We don't run this branch on CORE